### DISCUSSION TITLE
Can generated process visualization pages be integrated in a ReactJS application ?

### DISCUSSION THREAD

**Discussion by cockpittn** [frangoteam/FUXA#842] [Q&A]
Thank you for this great application.
I have a React JS application, and I want to integrate generated process pages into it. For now, I don't need to include the editor page.
Can it be done without using an iFrame ? I want to be able to manipulate the svg compoments : colors, animations, values, ...
Thank you for your support.

**Comment by unocelli** [comment]
Hi, Thanks for your interest...you mean without backend?

**Reply by cockpittn** [reply]
Yes, without backend, as long as the page will still running (MQTT, actions, animations, etc). But it will be interesting to customize some properties of the svg elements such as events or rendering.

**Reply by unocelli** [reply]
We have planned to create independent web components for the editor and the view that can communicate via a kind of bridge, but this is not a priority for the moment...help would be welcome 😉

**Reply by cockpittn** [reply]
Yes, it will be a great idea. I hope I can help you in it :)

**Reply by unocelli** [reply]
Great, we can try...I create a branch [wc-editor](https://github.com/frangoteam/FUXA/tree/wc-editor)
